#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_drraw_cgi.dpatch by Mohammad Ebrahim Mohammadi Panah <ebrahim@mohammadi.ir>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch for drraw.cgi

@DPATCH@

--- drraw-2.1.3/drraw.cgi	2008-12-15 23:57:22.000000000 +0330
+++ drraw-2.1.3.patched/drraw.cgi	2008-12-15 23:31:06.000000000 +0330
@@ -1,4 +1,4 @@
-#! /usr/local/bin/perl -T
+#! /usr/bin/perl -T
 #
 # Copyright (C) 2002, 2003, 2004, 2005 Christophe Kalt
 #
@@ -49,7 +49,7 @@
 
 # The configuration file is expected to be found in the same directory
 # as drraw itself.  You may customize this to be elsewhere.
-my $config = (dirname($0) =~ /(.*)/)[0] . "/drraw.conf"; # Untaint
+my $config = "/etc/drraw/drraw.conf"; # Untaint
 
 ###############################################################################
 ##   STOP STOP STOP STOP STOP STOP STOP STOP STOP STOP STOP STOP STOP STOP   ##
